docs/format_examples_datastore.py

```diff
@@ -0,0 +1,204 @@
+import pooch
+from pathlib import Path
+
+EXAMPLES_FOLDER = Path('format_examples')
+
+format_examples = [
+    dict(
```

Comment:
From a priority order, this should either present alphabetically OR we should list what we thing are the most common first (maybe tiff / svs / ome-tiff / ndpi / zarr / nd2 / czi / dicom / (others)).

Reply:
Reordered in [22b5b50](https://github.com/girder/large_image/pull/1622/commits/22b5b5050f8768ad27e724693d2adc4d92955d53)